### ISSUE TITLE
🚪 Add feature to export dashboard configuration to a json file

### ISSUE THREAD

**Issue by yodamad** [michelin/suricate#888]
## Problem

When migrating a suricate instance to another cluster, 

## Suggestion

Add a feature to export dashboard to backup, share or migrate configuration

**Comment by loicgreffier** [comment]
@yodamad 

There's an endpoint that exports (and imports) the whole configuration of an instance: https://github.com/michelin/suricate/blob/master/src/main/java/com/michelin/suricate/controller/ImportExportController.java#L76

Does this issue intend to export the configuration of a single dashboard?

**Comment by yodamad** [comment]
@loicgreffier 

Yes I'd need to export just a single dashboard to be able to provide a catalog of predefined dashboards in my company
